### PR TITLE
refactor code

### PR DESCRIPTION
**Pull request by arynn-gupta**

### CODE REVIEW

File: entry/src/main/js/default/pages/index/index.js [hunk 1]
```diff
@@ -127,4 +140,75 @@ export default {
...
+                console.log("fail data:"+ JSON.stringify(err));
+                console.log("fail code:"+ code)
+            },
+            complete: ()=>{
+                this.temperatureInCelsius = data.currentConditions.temp.c;
+                this.weather = data.currentConditions.comment;
+            }
+        })
+        const date = new Date();
+        const monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun",
```
Comment (archanaatr) [inline entry/src/main/js/default/pages/index/index.js:203]: please take the values of these arrays from locale string files

**Review by archanaatr — COMMENTED** [review]
pls check the comments.

File: entry/src/main/js/default/pages/index/index.js (hunk 1 above)
Comment (arynn-gupta) [reply entry/src/main/js/default/pages/index/index.js:203]: Actually this is intentional and works for both chinese and english language.
To get the name of current month :

1. We use `date.getMonth()` which returns the current month in number i.e. 0, 1, 2 and so on.
2. We use this number to get the corresponding month name from `monthNames` array.
3. Then we convert it to chinese or english accordingly using `this.$t('strings.'+monthNames[date.getMonth()])`

**Review by archanaatr — APPROVED** [review]
Ok

**Pull request merged**